Source/WebCore/loader/FrameLoader.h

```diff
@@ -352,6 +352,8 @@ class FrameLoader final : public CanMakeCheckedPtr<FrameLoader> {
     void loadItem(HistoryItem&, HistoryItem* fromItem, FrameLoadType, ShouldTreatAsContinuingLoad);
     HistoryItem* requestedHistoryItem() const { return m_requestedHistoryItem.get(); }
 
+    void updateURLAndHistory(const URL&, RefPtr<SerializedScriptValue>&& stateObject, NavigationHistoryBehavior historyHandling = NavigationHistoryBehavior::Replace);
```

Comment:
Please remove the unnecessary argument name `historyHandling`.

Reply:
Good point, I think the upload may have missed giving a warning about it. Fixed.